main.tf

```diff
@@ -15,6 +15,7 @@ resource "azurerm_template_deployment" "custom_alert" {
   parameters = {
     alertName                = "${var.alert_name}"
     alertDesc                = "${var.alert_desc}"
+    alertEnabled             = "${var.enabled}"
```

Comment:
What do you think about not creating the alert if it's disabled?
i.e. do a count = 0 on the template resource

Reply:
forgot about that feature. let me update the PR then